docs/source/topics/proc_deploying-sample-application-with-odo.adoc

```diff
@@ -0,0 +1,60 @@
+[id="deploying-sample-application-with-odo_{context}"]
+= Deploying a sample application with `odo`
+
+You can use OpenShift Do ([command]`odo`) to create OpenShift projects and applications from the command line.
+This procedure deploys a sample Java application to the OpenShift cluster running in the {prod} virtual machine.
+
+.Prerequisites
+
+* [command]`odo` is installed.
+For more information, see link:{odo-docs-url-installing}[Installing `odo`] in the [command]`odo` documentation.
+* The {prod} virtual machine is running.
+For more information, see <<starting-the-virtual-machine_{context}>>.
+* You must log in through [command]`oc` as the `developer` user.
+For more information, see <<accessing-the-openshift-cluster-with-oc_{context}>>.
+
+.Procedure
+
+To deploy a sample Java application through [command]`odo`, follow these steps:
+
+. Create a project for your application:
++
+[subs="+quotes,attributes"]
+----
+$ odo project create java-sample
+----
+
+. Create a directory for your components:
++
+[subs="+quotes,attributes"]
+----
+$ mkdir java-sample
+$ cd java-sample
+----
+
+. Create a Java component from a sample application on GitHub:
++
+[subs="+quotes,attributes"]
+----
+$ odo create java --git https://github.com/spring-projects/spring-petclinic.git
+----
+
+. Create a URL and add an entry to the local configuration file:
++
+[subs="+quotes,attributes"]
+----
+$ odo url create --port 8080
+----
+
+. Push the changes:
++
+[subs="+quotes,attributes"]
+----
+$ odo push
+----
++
+Your component is now deployed to the cluster with an accessible URL.
```

Comment:
Suggestion: add some verification step. It could be `odo url list` - it displays the URL(s) of the component or `odo list` - it displays all of the components of the application OR, instead of an extra command, an output of `odo push`, it should be something like

```Validation
 ✓  Checking component [7ms]

 Configuration changes
 ✓  Initializing component
 ✓  Creating component [134ms]

 Applying URL changes
 ✓  URL myurl: <URL HERE>

 Pushing to component <component-name> of type local
 ✓  Checking files for pushing [657850ns]
 ✓  Waiting for component to start [6s]
 ✓  Syncing files to the component [408ms]
 ✓  Building component [7s]
 ✓  Changes successfully pushed to component
```
I can quickly test it and give you the exact output with this app if needed.

Reply:
Fair enough -- originally, I included the `odo url list` step in the procedure, but noticed that the output of `odo push` prominently displayed the URL.

Personally, I'd like to avoid using output from commands in the documentation -- the commands themselves are unlikely to change, but the same can't be said for the output.

I'll go ahead and add the `odo url list` step back into the procedure just to be sure. :)

Reply:
Updated. Thank you!